packages/matplotlib/patches/disable_macosx_backend.patch

```diff
@@ -12,4 +13,5 @@ diff --git a/setupext.py b/setupext.py
              raise Skipped("Mac OS-X only")
```

Comment:
Why do we need this patch? Surely `sys.platform != 'darwin'`?

Reply:
"This comes from #541. We let it as is in case it helps building on Mac."

I don't know if it helps on a Mac, I'm running in docker. Probably should try running native at some point. What does this return when cross compiling?

Reply:
I've run it native before. I guess that's the issue that it returns Darwin on macs. The Python ecosystem is really bad at cross compiling =(